.github/workflows/sync-api-vmlinux-cron.yaml

```diff
@@ -0,0 +1,99 @@
+name: Sync api.h and vmlinux.h from latest kernel
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 1 * *' # first day of each month
+
+jobs:
+  sync-api-vmlinux:
+    runs-on: ${{ (matrix.arch == 'arm64' && 'ubuntu-24.04-arm') || 'ubuntu-24.04' }}
```

Comment:
While i could do everything in a single x86_64 ubuntu job, i chose to run it in parallel on both architectures and then have a single job collect built headers and create a PR; this way the permissions to write pull_requests are only available to a small scope job.